osf/models/node.py

```diff
@@ -902,12 +902,34 @@ def get_admin_contributor_ids(node):
             admin_ids.update(set(admins).difference(contributor_ids))
         return admin_ids
 
+    @property
+    def parent_admin_contributor_ids(self):
```

Comment:
I think these properties could be a bit DRYer. You could add a helper function with the duped code and each property could just call that function with the appropriate params. Something like this (extremely rough pseudocode where my logic is probably backwards, but you get the idea):
```
def _get_admin_contributor_ids(node, include_self=False):
    def get_admin_contributor_ids(node):
        return ...
    contributor_ids = ...
    admin_ids = set() if not include_self else set(get_admin_contributor_ids(node)
    ...

@property
def admin_contributor_ids(self):
    return _get_admin_contributor_ids(self, include_self=True)

@property
def parent_admin_contributor_ids(self):
    return _get_admin_contributor_ids(self, include_self=True)
```